repose-aggregator/commons/commons-utilities/src/main/scala/org/openrepose/commons/utils/servlet/http/HttpServletRequestWrapper.scala

```diff
@@ -145,4 +151,71 @@ class HttpServletRequestWrapper(originalRequest: HttpServletRequest, inputStream
   def getSplittableHeaderScala(headerName: String): List[String] = getHeadersScala(headerName).foldLeft(List.empty[String])((list, s) => list ++ s.split(","))
 
   override def getSplittableHeaders(headerName: String): util.List[String] = getSplittableHeaderScala(headerName).asJava
+
+  /**
+    * @return a [[StringBuffer]] containing the reconstructed URL for this request (note that
+    *         mutation of this [[StringBuffer]] will have no effect on the request URL)
```

Comment:
wait it doesn't return them? that's super weird. forget it then. mark it as a known oddity